src/libraries/System.Private.Xml/src/System/Xml/Core/XmlReader.cs

```diff
@@ -601,10 +618,10 @@ public virtual object ReadElementContentAs(Type returnType, IXmlNamespaceResolve
         public abstract int AttributeCount { get; }
 
         // Gets the value of the attribute with the specified Name
-        public abstract string GetAttribute(string name);
+        public abstract string? GetAttribute(string name);
 
         // Gets the value of the attribute with the LocalName and NamespaceURI
-        public abstract string GetAttribute(string name, string namespaceURI);
+        public abstract string? GetAttribute(string name, string namespaceURI);
 
         // Gets the value of the attribute with the specified index.
         public abstract string GetAttribute(int i);
```

Comment:
Seems also  can return null

Reply:
I think this will be non-null or you'll get exception (you're asking for specific property). I didn't get any problems with this annotation on the inherited classes yet and I'd expect to if that was incorrect